woocommerce/woocommerce-gutenberg-products-block
# Reformatted prettier files

A lot of people have been having problems with `prettier` errors on the pre-commit hook. I ran `npm run reformat-files` and it looks like a few files weren't formatted properly so I've committed the formatting changes here. 


It would  be useful if a few people could check this to make sure i doesn't cause any problems for anyone. 

To test: 
1. Checkout the branch
2. Run `npm install`
3. Make a commit anywhere in `js/assets` folder.
4. You should not get any prettier or es-lint errors.
